docs/notebooks/Translation_Matrix_Revisit.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
Please add this test to `translation_matrix.ipynb` (as another section) and remove  `Translation_Matrix_Revisit.ipynb`